core/backend/src/rpc-impl/IModelReadRpcImpl.ts

```diff
@@ -31,20 +31,32 @@ export class IModelReadRpcImpl extends RpcInterface implements IModelReadRpcInte
   public static register() { RpcManager.registerImpl(IModelReadRpcInterface, IModelReadRpcImpl); }
 
   public async getConnectionProps(tokenProps: IModelRpcOpenProps): Promise<IModelConnectionProps> {
-    return RpcBriefcaseUtility.openWithTimeout(RpcTrace.currentActivity!, tokenProps, SyncMode.FixedVersion);
+    if (undefined === RpcTrace.currentActivity) {
```

Comment:
If you are inside an RpcImpl method, you are guaranteed that `RpcTrace.currentActivity` is defined. `assert` and move on.